doc/tag_filter.in

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+target_file="${1}"
+
+if [[ -z "${target_file}" ]];
+then
+    exit
+fi
+
+py_exec="@PYTHON_EXECUTABLE@"
+py_sh="@FreeOrion_SOURCE_DIR@/doc/tag_filter.py"
+
+
+if [[ -r "${target_file}" && -x "${py_exec}" && -r "${py_sh}" ]];
+then
+    eval "$py_exec $py_sh $target_file"
```

Comment:
Doxygen,  I will explain more in a comment outside of this review block (so others may see it easier)